Data Structures/Array/unique_element

```diff
@@ -0,0 +1,20 @@
+package gssoc;
+
+public class Dublicateinarray {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+
+		int[] arr = { 5, 8, 9, 5, 9, 7, 8 };
```

Comment:
Take input from the user. 

Also solve this: Given an array in which all numbers except two are repeated once. Find those two numbers in the most efficient way.

Reply:
O(2n) okay?

Reply:
@ramitsawhney27 Could you kindly review this PR again as suggested changes were made?